EPCPortalTesting/ReportDataModelTests.cs

```diff
@@ -0,0 +1,124 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using EPCPortalWeb;
+using EPCPortalWeb.Models;
+using System.Collections.Generic;
+
+namespace EPCPortalTesting
+{
+    [TestClass]
+    public class ReportDataModelTests
+    {
+        ReportDataModel reportDataModel = new ReportDataModel();
+        
+        [TestMethod]
+        public void CompareCurrentAndPotentialNumericalValues_WhereCurrentIsGreaterThanPotential_ForHeating()
```

Comment:
Like the use of Where to help set up test, only missing the pass condition at end, i.e., _HeatingRecommendationsIsNotNull

Same comment applies to other tests in this class.